app/code/core/Mage/Adminhtml/controllers/Sales/OrderController.php

```diff
@@ -683,24 +683,21 @@ protected function _isAllowed()
      */
     public function exportCsvAction()
     {
-        $fileName   = 'orders.csv';
-        $grid       = $this->getLayout()->createBlock('adminhtml/sales_order_grid');
-        $this->_prepareDownloadResponse($fileName, $grid->getCsvFile());
+        $grid = $this->getLayout()->createBlock('adminhtml/sales_order_grid');
+        $this->_prepareDownloadResponse(...$grid->getCsvFile('orders.csv', 1000));
```

Comment:
```suggestion
        $this->_prepareDownloadResponse(...$grid->getCsvFile('orders.csv', -1));
```

shouldn't this be -1 like the rest of the calls?